### DISCUSSION TITLE
[docker log] /var/empty must be owned by root and not group or world-writable.

### DISCUSSION THREAD

**Discussion by JivinDotL** [gogs/gogs#7237] [Q&A]
I used the latest at 2022.11.8,  and I found in the dockor log showing [/var/empty must be owned by root and not group or world-writable.]

docker pull gogs/gogs:latest

pls help to check this issue. thanks.

**Comment by JivinDotL** [comment]
fix it via bellow command.
chmod 744 /var/empty/

**Reply by bburgy** [reply]
But this is not a fix, this is a workaround. The permission should be well configured in the image, not in the container?
